tools/c7n_index/c7n_index/metrics.py

```diff
@@ -47,14 +51,15 @@
 
 CONFIG_SCHEMA = {
     'type': 'object',
-    'additionalProperties': True,
-    'required': ['indexer', 'accounts'],
```

Comment:
If we make `accounts` optional I think we'll need to change the code here
https://github.com/salwandhruvdev/cloud-custodian/blob/e62d4f555f4234f596c8d92820c46b5cae9b45eb/tools/c7n_index/c7n_index/metrics.py#L483
and here
https://github.com/salwandhruvdev/cloud-custodian/blob/e62d4f555f4234f596c8d92820c46b5cae9b45eb/tools/c7n_index/c7n_index/metrics.py#L569
to `for account in config.get('accounts', ()):`

Reply:
Since `accounts` is not used by `sqs-indexer`, I have added validation for this key in their respective callers.
https://github.com/salwandhruvdev/cloud-custodian/blob/incoming-salwan-add-sqs-consumer/tools/c7n_index/c7n_index/metrics.py#L460-L464